src/interfaces/author.ts

```diff
@@ -1,27 +1,28 @@
-import { OtherTags } from './other-tags';
+interface ImageResponse {
+    edges: { node: { url: string } }[];
+}
 
 export type Image = {
     url: string;
 };
-
-type ArticlesByAuthor = {
-    name: string;
-    calculated_slug: string;
-    description: string;
-    otherTags: OtherTags[];
-};
-
-export interface Author {
-    name: string;
+interface AuthorBase {
     url?: string;
     bio?: string;
-    image?: Image;
     location?: string;
-    title?: string;
     linkedin?: string;
     facebook?: string;
     twitter?: string;
     youtube?: string;
-    articles?: ArticlesByAuthor[];
     calculated_slug: string;
 }
+export interface Author extends AuthorBase {
+    image?: Image;
+    title?: string;
+    name: string;
+}
+
+export interface AuthorResponse extends AuthorBase {
+    imageConnection?: ImageResponse;
+    job_title?: string;
+    title: string;
+}
```

Comment:
Had to separate these two interfaces, since the fields in CS are differently named than the fields in Strapi. Also, I've removed `articles` here because we don't really need it. All content items already reference an author, so we can just filter the content items preval to find all content by a particular author. See [here](https://github.com/mongodb/devcenter/pull/483/files#diff-c5a9b76b1e3eb322d6e1d7f6601467d4cf9fc6eff73249f5cf26dc2d5114ad1fR264-R269).